### PR TITLE
Awesome book es6

### PR DESCRIPTION
**Pull request by shubhambhoyar077**
### Awesome Book

**Project Requirements:-**

- Used ES6 syntax.

- Divide your code into modules.
- Refactored functions to arrow functions.

- Use let and const in the correct way.

- For Date and Time using a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  /* justify-content: center; */
+  padding: 2% 7%;
+
+  /* max-width: 1100px; */
```
Comment (AmaduKamara) [inline css/index.css:14]: - [ ] It's a good convention to not have commented codes in our codebase as they make our codebase dirty and not professional. Kindly remove all commented codes from your codebase so your code looks clean and professional 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @shubhambhoyar077,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used ES6 modules, syntax, and arrow function 👍🏽
✅ Add and remove features work fine 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: css/index.css (hunk 1 above)
Comment (shubhambhoyar077) [reply css/index.css]: Comments removed

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 5px;
+  margin-top: 5%;
+
+  /* width: 40%; */
+}
```
[the remark this answers is not in the document]
Comment (shubhambhoyar077) [reply css/index.css]: Comments removed

**Pull request merged**